docs/spec.md

```diff
@@ -1344,14 +1344,14 @@ in the `operand` tensor and produces a `result` tensor. More formally,
 
 | Label | Name                   | Type                                         | Constraints   |
 |-------|------------------------|----------------------------------------------|---------------|
-| (I1)  | `operand`              | tensor                                       | (C1-C3), (C5) |
+| (I1)  | `operand`              | tensor or per-tensor quantized tensor        | (C1-C2), (C5) |
```

Comment:
why do we need to support only per tensor quantization here? When adding bias value and the result of quantized convolution, they might need to add broadcast_in_dim op if the shapes of two tensors are not the exactly same.

Reply:
Hi @abattery Thanks for the comment and the information.
We have a separate issue https://github.com/openxla/stablehlo/issues/1574 just to to figure out the story for per-axis quantization: where we are seeking for some clear criterion on supporting per-axis scheme for StableHLO ops. 

>> When adding bias value and the result of quantized convolution, they might need to add broadcast_in_dim op if the shapes of two tensors are not the exactly same.

From the above it seems that both  `broadcast_in_dim` and `add` could be candidates for supporting per-axis scheme. 
@abattery Do you mind help further the discussion in #1574 with your opinion? For example, could there be other candidates?

Reply:
While I think there might be a use case when we have a scalar bias (scalar in terms of storage type) in the convolution:

```
%conv = "stablehlo.convolution"(%input, %weight)
%bias_scalar_cst = "stablehlo.constant"() {value = dense<1> : tensor<i32>}
    : tensor<!quant.uniform<ui8:f32, 1.0:0>>
%bias = "stablehlo.broadcast_in_dim"(%bias_scalar_cst)
    {broadcast_dimensions = [0]}
    : (tensor<!quant.uniform<ui8:f32, 1.0:0>>)
    -> tensor<4x!quant.uniform<ui8:f32:0, 1.0:0,2.0:0,3.0:0,4.0:0>> 
%add = "stabehlo.add"(%conv, %bias) : ...
```

However, we will have to make it explicit that the broadcast operates on storage type.  In this example, it means we broadcast the storage type value `1` into 4 elements.  And their corresponding dequant values will be `[1.0, 2.0, 3.0, 4.0]`.  Also, we might have some problem in output type inference (since per-axis quantization parameters are unknowns) though this problem is nothing new to `broadcast_in_dim` (since the output shape is unknown too).

OTOH, for this particular case, I think it will make more sense to utilize the `isScalar` in the `DenseElementsAttr`, i.e. the constant can be specified as:

```
%conv = "stablehlo.convolution"(%input, %weight)
%bias = "stablehlo.constant"() {value = dense<1> : tensor<4xi32>}
    :tensor<4x!quant.uniform<ui8:f32:0, 1.0:0,2.0:0,3.0:0,4.0:0>>
%add = "stabehlo.add"(%conv, %bias) : ...
```

Reply:
Thanks Logan for the detailed example. Also I'm fine with the discussion separately.